utils/variables_computation/jet_variables_helper.py

```diff
@@ -0,0 +1,51 @@
+import numba as nb
```

Comment:
What about adding this to `jet_variables`?
Makes little sense to have two separate scripts I think! It would make sense if `jet_variables` was a user configurable code, but both scripts are hidden to the user, and only used for each other...

Reply:
removed jet_variables_helper.py, moved necessary functions to jet_variables